maestro-client/src/main/java/maestro/drivers/IOSDriver.kt

```diff
@@ -251,20 +251,20 @@ class IOSDriver(
         when (swipeDirection) {
             SwipeDirection.UP -> {
                 iosDevice.scroll(
-                    xStart = width / 4,
-                    yStart = height,
-                    xEnd = width / 4,
-                    yEnd = height / 4,
+                    xStart = width / 2,
```

Comment:
@dmitry-zaitsev I think we need to use the grid here also right?

Reply:
Correct